src/js/contentLibrary.js

```diff
@@ -0,0 +1,47 @@
+import { modalWindow } from './getModalWindowAPI';
```

Comment:
All rest requests that retrieves movies should be located in one file , for example in MovieApiService (get one movie , get all movies, get trending movies)

Reply:
And all classes should be capitalized

Reply:
edit all classses API and thier names